sct@skulptist.de/README.md

```diff
@@ -0,0 +1,29 @@
+# SCT Toggle 
+
+A Linux Mint Cinnamon Applet to toggle through color temperatures.
```

Comment:
Just a comment, Cinnamon Spices are distro-independent.

Reply:
Oh, you are right.
Are they tied to Cinnamon or are there other runtimes that use the spices?

Reply:
By default, the code is part of Cinnamon itself. Distros could modify this if they wanted to of course but I don't know of any that do.

As far as compatibility, only the Themes could be ported or compatible elsewhere depending on how they are structured. The recently published Actions can be used anywhere with Nemo as well, Cinnamon or not.

Reply:
Ok, I will modify that part in the next version.

Reply:
No rush! 😅